FirebaseAuth/Tests/Unit/FIRSetAccountInfoRequestTests.m

```diff
@@ -174,6 +174,16 @@
  */
 static NSString *const kTestDeleteProviders = @"TestDeleteProviders";
 
+/** @var kDeleteProvidersKey
+    @brief The key for the "deleteProvider" value in the request.
+ */
+static NSString *const kDeletePasskeysKey = @"deletePasskey";
+
+/** @var kTestDeleteProviders
+    @brief The fake @c deleteProviders for testing the request.
```

Comment:
nit: deletePasskeys

Reply:
Cannot update it since this is the  matches with the request proto.